src/backend/aspen/workflows/nextstrain_run/build_plugins/base_plugin.py

```diff
@@ -16,7 +16,7 @@ def __init__(
         self.template_args = template_args
         self.template = None
         self.tree_build_level = "location"
-        # Set self.num_county_sequences, self.num_sequences, etc.
```

Comment:
`num_county_sequences` had been used as an example, but it's no longer anywhere in our code base, so I removed since I don't think it makes for a very good example any more.